src/main/java/com/gmail/goosius/siegewar/settings/ConfigNodes.java

```diff
@@ -603,7 +603,38 @@
 			"true",
 			"",
 			"# If this value is true, then a town under occupation cannot unclaim.",
-			"#  This setting is recommended, to avoid occupation escape exploits.");	
+			"#  This setting is recommended, to avoid occupation escape exploits."),
+
+	PUNISH_NON_SIEGE_PARTICIPANTS_IN_SIEGE_ZONE(
+			"punish_non_siege_participants_in_siege_zone",
+			"",
+			"############################################################",
+			"# +------------------------------------------------------+ #",
+			"# |                     War Sickness                     | #",
+			"# +------------------------------------------------------+ #",
+			"############################################################",
+			""),
+
+	ENABLE_SICKNESS(
+			"punish_non_siege_participants_in_siege_zone.enable_sickness",
+			"true",

```

Comment:
:construction: 
- As per previous comment, please set this default to "false"
- Individual servers can then enable it if they like the feature etc.